packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -59,10 +65,25 @@ const MediaReplaceFlow = ( {
 		}
 	};
 
+	const onError = ( message ) => {
+		const renderMsg = renderToString( message );
+		const speakMsg = debounce(
+			() => speak( renderToString( message ) ),
+			5000
+		);
+		speakMsg();
+		createNotice( 'error', renderMsg, {
+			id: errorNoticeID,
+			isDismissible: true,
+			__unstableHTML: true,
```

Comment:
We should really try to avoid this. `__unstableHTML` ([by its name](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis)) should be assumed to be targeted for removal, and we don't want to perpetuate its existence. Its implementation using `dangerouslySetInnerHTML` can be problematic in light of [inherent risks therein](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml).

Ultimately, we want the API interface of `createNotice` to serve whatever requirements one might think to resort to `__unstableHTML`. What are those requirements in this case? Do we even need it?

Reply:
Yes, good call @aduth , I've removed the need to use `__unstableHTML` by stripping the tags from the error message,